reactor-core/src/test/java/reactor/core/publisher/SerializedSubscriberTest.java

```diff
@@ -90,6 +90,9 @@ public void testLeakWithRetryWhenImmediatelyCancelled() throws InterruptedExcept
 		AtomicInteger seen = new AtomicInteger();
 
 		final CountDownLatch latch = new CountDownLatch(1);
+		final CountDownLatch latch1 = new CountDownLatch(1);
```

Comment:
why not just changing the `latch` to `new CountDownLatch(4)` and output the current value if `await` fails?

Reply:
yeah. fixed